### PR TITLE
fix: various invite flow fixes & improvements

### PR DESCRIPTION
**Pull request by fabis94**
https://linear.app/speckle/issue/WEB-1142/project-invite-to-a-private-project-403
https://linear.app/speckle/issue/WEB-1143/using-project-invite-to-sign-up-fails
https://linear.app/speckle/issue/WEB-1144/if-project-invite-show-signup-cta

### CODE REVIEW

**Comment by fabis94** [comment]
New error page if you don't have access to the project: 
image

New error page if you don't have access, but have an active invite:
image

File: packages/frontend-2/components/projects/invite/Banner.vue [hunk 1]
```diff
@@ -77,13 +95,41 @@ const { isLoggedIn } = useActiveUser()
...
+})
+
+const mainInfoBlockClasses = computed(() => {
+  const classParts = ['flex grow']
+
+  if (props.block) {
+    classParts.push('flex-col space-y-2 items-center')
+  } else {
+    classParts.push('flex-row space-x-2 items-center text-sm')
+  }
```
Comment (andrewwallacespeckle) [inline packages/frontend-2/components/projects/invite/Banner.vue:127]: Really minor, but items-center can be moved to classParts

**Review by andrewwallacespeckle — APPROVED** [review]
Looks good, apart from the minor comment I left.

All these tickets were originally raised by @gjedlicska, so it's probably worth waiting for him to check this too.

**Review by gjedlicska — APPROVED** [review]
functionality looks good

**Pull request merged**